SeleniumWebDriverBasics/SeleniumWebDriverBasics/SpecFlow/Steps/YandexUnitTestsSteps.cs

```diff
@@ -0,0 +1,207 @@
+using SeleniumWebDriverBasics.Entities;
+using SeleniumWebDriverBasics.Utilities;
+using SeleniumWebDriverBasics.WebDriver;
+using SeleniumWebDriverBasics.WebObjects.Pages;
+using TechTalk.SpecFlow;
+
+namespace SeleniumWebDriverBasics.SpecFlow.Steps
+{
+    [Binding]
+    public sealed class YandexUnitTestsSteps
+    {
+        private readonly ScenarioContext senarioContext;
+        private readonly LoginPage loginPage = new LoginPage();
+        private readonly EmailPage emailPage = new EmailPage();
+        private readonly HomePage homePage = new HomePage();
+
+        public YandexUnitTestsSteps(ScenarioContext senarioContext)
+        {
+            this.senarioContext = senarioContext;
+        }
+
+        [Given(@"I log in to yandex mail box")]
+        public void TestSetup()
+        {
+            Browser.GetInstance();
+            Browser.NavigateTo();
+            Browser.MaximizeWindow();
+
+            var login = Convert.ToString(Configuration.Login);
+            var password = Convert.ToString(Configuration.Password);
+            var user = new User(login, password);
+
+            homePage.ClickOnLoginButton();
+            loginPage.Login(user);
+            homePage.WaitForComposeLinkIsVisible();
+        }
+
+        private string actualAccountName;
+
+        [When(@"I get the actual user name from User Account option")]
+        public string GetActualAccountName()
+        {
+            actualAccountName = emailPage.GetActualUserName();
+            return actualAccountName;
+        }
+
+        [Then(@"the actual user name should match the entered login")]
+        public void ActualAndExpectedUserNameComparison()
```

Comment:
A method's name should be an action. I suggest to rename "ActualAndExpectedUserNameComparison" to "CompareActualAndExpectedUserName".